docs/internal/investigations/remote-runtime-configuration-poc/poc/README.md

```diff
@@ -0,0 +1,140 @@
+# Dapr Proof of Concept scenario
```

Comment:
I would suggest removing the additional `poc` directory under `remote-runtime-configuration-poc` and moving the Runtime Governor Concept document (currently located in `remote-runtime-configuration-poc/README.md`) to the root of investigations.


Anyway, if that will be easier I can do that in my PR https://github.com/kyma-project/control-plane/pull/68/files after resolving some conflicts.